app/views/tasks/_form.html.slim

```diff
@@ -83,11 +83,11 @@
         /.field-element
           = f.label :labels, t('exercises.labels'), class: 'form-label'
           = f.select :labels, options_for_select(Label.all.collect{ |l| [l.name, l.name] }, @task.labels.pluck(:name)), {}, {class: "my-tag form-control", multiple: "multiple", style: "width: 300px;"}
-        /.field-element
+        .field-element
           = f.label :license, t('exercises.license'), class: 'form-label'
           .license
             .select
-              = f.select :license_id, options_for_select(License.all.collect{ |l| [l.name, l.id, {'data-link'=> l.link}] }, @license_default), {}, {class: 'form-control', disabled: @license_hidden}
+              = f.select :license_id, options_for_select(License.all.collect{ |l| [l.name, l.id, {'data-link'=> l.link}] }, @task.license_id), {include_blank: t('exercises.form.none')}, {class: 'form-control'}
```

Comment:
Would you mind updating the translation key to something below the `task` prefix? I know, it's still not consistent, but I would propose to move everything away from `exercise` to `task`. This will make it more consistent and potentially allow so-called [lazy lookups](https://guides.rubyonrails.org/i18n.html#lazy-lookup). Personally, I am not a fan of lazy lookups, as they could make it more difficult to move translations around, but that's another story ;).

If you move and change the locale, one could also consider introducing an explicit / new translation: What if we define no license chosen as "All rights reserved"? I am not a legal expert, but would argue that you don't gain permission in case someone doesn't provide an explicit license stating otherwise?

Reply:
I think not selecting a license should not result in "All rights reserved" being shown as a license. Just showing that no license was selected is most accurate in my opinion.

Reply:
That's true. Let's keep it this way and allow all users to be aware of a missing, explicit license.